src/common/SmartNumericInput/SmartNumericInput.tsx

```diff
@@ -8,6 +8,7 @@ export interface SmartNumericInputProps {
     defaultValue: number;
     min: number;
     max: number;
+    originalValue?: string;
```

Comment:
Why is this needed / used?

Reply:
It is needed to keep track of the value we are comparing against. But not all smart numeric inputs are election settings, which is why not all smart numeric inputs are supposed to have an original value.